demisto_sdk/commands/find_dependencies/tests/find_dependencies_test.py

```diff
@@ -678,6 +689,89 @@ def test_collect_playbooks_dependencies_on_incident_fields(self, id_set):
 
         assert IsEqualFunctions.is_sets_equal(found_result, expected_result)
 
+    def test_collect_playbooks_dependencies_on_incident_fields__phishing_pack(self, id_set):
+        """
+        Given
+            - A playbook entry in the id_set.
```

Comment:
add more description here. because the reason for phishing to be optional is because `Search And Delete Emails - EWS` has a playbook or classifier that use a field from Phishing